### PR TITLE
Scope adjustment overhaul

### PR DESCRIPTION
**Pull request by esteldunedain**
Haven't tested this ingame yet. 

By default it uses the same keybinds than small adjustments + Shift

### CODE REVIEW

**Comment by ulteq** [comment]
It's working now.

File: addons/scopes/functions/fnc_canAdjustZero.sqf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*
+ * Author: KoffeinFlummi, Ruthberg
+ * Changes the adjustment for the current scope
```
Comment (esteldunedain) [inline addons/scopes/functions/fnc_canAdjustZero.sqf]: Wrong description

**Comment by ulteq** [comment]
Ready to be merged if there are no objections.

File: addons/scopes/functions/fnc_adjustScope.sqf [hunk 2]
```diff
@@ -1,56 +1,71 @@
...
-} else {
+if (_majorStep) then {
+	switch (_turretAndDirection) do
+	{
+		case ELEVATION_UP:   { _elevation = ceil(_elevation) };
+		case ELEVATION_DOWN: { _elevation = floor(_elevation) };
+		case WINDAGE_LEFT:   { _windage = floor(_windage) };
+		case WINDAGE_RIGHT:  { _windage = ceil(_windage) };
+	};
+};
```
Comment (KoffeinFlummi) [inline addons/scopes/functions/fnc_adjustScope.sqf]: > Ready to be merged if there are no objections.

Taaaaaabbbbs....

**Pull request merged**